composer.json

```diff
@@ -27,13 +27,14 @@
         }
     },
     "require": {
-        "php": "^7.3 || ~8.0.0",
+        "php": "^7.4 || ~8.0.0 || ~8.1.0",
         "laminas/laminas-zendframework-bridge": "^1.0"
     },
     "require-dev": {
         "doctrine/migrations": "^2.0 || ^3.0",
         "enlightn/security-checker": "^1.2",
         "guzzlehttp/guzzle": "^5.3.3 || ^6.3.3",
+        "laminas/laminas-code": "4.5.x-dev#fbdaecda0c91ea0c7aa81c89eda16b2f345d0657 as 4.5.0",
```

Comment:
This should be dropped before merging

Reply:
@Ocramius You mean whole line, or just commit sha? The problem is that laminas-code 4.4.* is not supporting PHP 8.1 and 4.5 tag is not pushed yet.

Reply:
Correct - whole line should be removed. Well aware that `laminas-code` does not support `php: 8.1` yet.

Reply:
Done